apps/indexer/ponder.schema.ts

```diff
@@ -94,7 +96,7 @@ export const votingPowerHistory = onchainTable(
 export const delegation = onchainTable(
   "delegations",
   (drizzle) => ({
-    transactionHash: drizzle.text("transaction_hash").notNull(),
+    transactionHash: drizzle.text("transaction_hash"),
     daoId: drizzle.text("dao_id").notNull(),
     delegateAccountId: drizzle.text("delegate_account_id"),
```

Comment:
The primary key definition is missing the transactionHash column. Since transactionHash is now nullable but still part of the primary key array, this will cause database constraint violations when transactionHash is null.